ui/apps/platform/src/routePaths.js

```diff
@@ -11,8 +11,6 @@ export const authResponsePrefix = '/auth/response/';
 export const authorizeRoxctlPath = '/authorize-roxctl';
 
 export const dashboardPath = `${mainPath}/dashboard`;
-export const networkBasePath = `${mainPath}/network`;
-export const networkPath = `${networkBasePath}/:deploymentId?/:externalType?`;
 export const networkBasePathPF = `${mainPath}/network-graph`;
```

Comment:
should we just rename this to `networkBasePath` since there is only one version of the network graph now?